pages/1.10/installing/ent/upgrading/index.md

```diff
@@ -190,7 +190,7 @@ Proceed with upgrading every master node one-at-a-time in any order using the fo
     1.  Wait until the `dcos-mesos-master` unit is up and running.
     1.  Verify that `curl http://<dcos_master_private_ip>:5050/metrics/snapshot` has the metric `registrar/log/recovered` with a value of `1`.
         **Tip:** If you are upgrading from permissive to strict mode, this URL will be `curl https://...` and you will need a JWT for access.
-    1.  Verify that `/opt/mesosphere/bin/mesos-master --version` indicates that the upgraded master is running Mesos 1.4.0.
+    1.  Verify that `/opt/mesosphere/mesos 1.4.2` indicates that the upgraded master is running Mesos 1.4.2.
```

Comment:
I don't think `/opt/mesosphere/mesos 1.4.2` is a thing? The previous command was correct. Why did you change the command?